sentry_sdk/client.py

```diff
@@ -249,15 +249,28 @@ def _capture_envelope(envelope):
 
             self.metrics_aggregator = None  # type: Optional[MetricsAggregator]
             experiments = self.options.get("_experiments", {})
-            if experiments.get("enable_metrics", True):
-                from sentry_sdk.metrics import MetricsAggregator
-
-                self.metrics_aggregator = MetricsAggregator(
-                    capture_func=_capture_envelope,
-                    enable_code_locations=bool(
-                        experiments.get("metric_code_locations", True)
-                    ),
-                )
+            if experiments.get("enable_metrics", True) or experiments.get(
+                "force_enable_metrics", False
+            ):
+                try:
+                    import uwsgi  # type: ignore
+                except ImportError:
+                    uwsgi = None
+
+                if uwsgi is not None and not experiments.get(
+                    "force_enable_metrics", False
+                ):
+                    logger.warning("Metrics currently not supported with uWSGI.")
+
+                else:
+                    from sentry_sdk.metrics import MetricsAggregator
+
+                    self.metrics_aggregator = MetricsAggregator(
+                        capture_func=_capture_envelope,
+                        enable_code_locations=bool(
+                            experiments.get("metric_code_locations", True)
+                        ),
+                    )
```

Comment:
Kinda a nitpick, but couldn't we get rid of this additional `if/else` branching simply by moving the `if` part to an `else` block associated with the try statement, and by moving the else portion under `except ImportError`? Something like this:
```suggestion
                try:
                    import uwsgi as _  # type: ignore
                except ImportError:
                    from sentry_sdk.metrics import MetricsAggregator

                    self.metrics_aggregator = MetricsAggregator(
                        capture_func=_capture_envelope,
                        enable_code_locations=bool(
                            experiments.get("metric_code_locations", True)
                        ),
                    )
                else:
                    if not experiments.get(
                        "force_enable_metrics", False
                    ):
                        logger.warning("Metrics currently not supported with uWSGI.")
```

Reply:
I'm pretty sure this would work but I'd have to recheck this locally first and I'm kinda eager to get this hotfix out :) This code will go back to how it was anyway as soon as we figure out an actual fix.

Reply:
Gotcha, fair enough. Does not matter if this change is only going to be temporary